pkg/agent/memberlist/cluster.go

```diff
@@ -221,7 +222,13 @@ func (c *Cluster) handleCreateNode(obj interface{}) {
 	if member, err := c.newClusterMember(node); err == nil {
 		_, err := c.mList.Join([]string{member})
 		if err != nil {
-			klog.ErrorS(err, "Processing Node CREATE event error, join cluster failed", "member", member)
+			// Formatting the error message.
+			errorMessage := err.Error()
+			prefixToRemove := "1 error occurred:"
+			errorMessage = strings.TrimPrefix(errorMessage, prefixToRemove)
+			errorMessage = strings.Replace(errorMessage, "\n\t* ", "", -1)
+			errorMessage = strings.TrimSpace(errorMessage)
```

Comment:
This is too specific to the error's format and content. It should handle the case more generically. 

Given that the type of the error returned by `c.mList.Join` is `github.com/hashicorp/go-multierror.Error`, and we know there could be at most 1 wrapped error (we only join 1 member in this case), we can just call `errors.Unwrap(err)` to get the wrapped error and use it in the log.